radar/shared/src/types.rs

```diff
@@ -32,3 +33,9 @@ pub struct RadarPlayerInfo {
     pub position: [f32; 3],
     pub rotation: f32,
 }
+
+#[derive(Serialize, Deserialize, Clone, Debug)]
+#[serde(rename_all = "camelCase")]
+pub struct RadarBombInfo {
```

Comment:
Maybe add a flag whatever it's planted or not.  
Maybe use a similar system as I've done here including the bomb state: https://github.com/Valthrun/Valthrun/blob/master/controller/src/enhancements/bomb.rs#L51

In your case it would be RadarBombInfo containing the position and the C4State which may includes `Dropped, Carried` as well

Reply:
I added more data into RadarBombInfo. You can check it out.